components/mediation-ui/org.wso2.carbon.rest.api.ui/src/main/resources/web/api/index.jsp

```diff
@@ -642,6 +651,32 @@
                     <% } %>
                 </div>
             </td>
+            <td width="20px" style="text-align:left;border-left:none;border-right:none;width:100px;">
+                <div class="inlineDiv">
+                    <% if (apiData.getArtifactContainerName() != null) { %>
+                    <a style="color:gray;background-image:url(images/favicon-16x16.png);" class="icon-link" href="#"
+                       onclick="#">swagger</a>
+                    <% } else {
+                        String url=null;
+                        if (tenantDomain != "carbon.super") {
+                            url = serverContext + "/t/" + tenantDomain + "/" + Encode.forHtmlAttribute(apiData.getName());
+                        } else {
+                            url = serverContext + "/" + Encode.forHtmlAttribute(apiData.getName());
+                        }
+                    %>
+                    <a style="background-image:url(images/favicon-16x16.png);" class="icon-link" href="<%=url + "?swagger.json"%>"
+                       onclick="#" %>swagger</a>
+                    <% } %>
+                </div>
+            </td>
+            <td width="20px" style="text-align:left;border-left:none;width:100px;">
+                <div class="inlineDiv">
+                    <a style="background-image:url(images/tryit.gif);" class="icon-link"
+                    href="swaggereditor.jsp?action=tryIt&apiName=<%=Encode.forHtmlAttribute(apiData.getName())%>"

```

Comment:
Should be opened in a new window and also, should open the TryIt UI which is accessible without logging in.

Example:

`<div class="inlineDiv">
                    <% if (apiData.getArtifactContainerName() != null) { %>
                    <a style="background-image:url(images/tryit.gif);" class="icon-link"
                       href="#"
                       onclick="#"><fmt:message key="api.tryit"/></a>
                    <% } else {
                        String url=null;
                        if (tenantDomain != "carbon.super") {
                            url = serverContext + "/t/" + tenantDomain + "/" + Encode.forHtmlAttribute(apiData.getName());
                        } else {
                            url = serverContext + "/" + Encode.forHtmlAttribute(apiData.getName());
                        }
                    %>
                    <a style="background-image:url(images/tryit.gif);" class="icon-link" href="<%=url + "?swaggertryit"%>"
                       onclick="#" target="_blank" %><fmt:message key="api.tryit"/></a>
                    <% } %>
                </div>`